File: Refresherquizmod1.py
import pandas as pd
data = pd.read_csv("/Users/gregwolcott/Documents/UNCW/BAN 502 - Predictive Analytics/Module 1/InventoryData.csv")

#1 number rows
print(len(data))
#2 number rows and columns
print(data.shape)
#3 Print the mean
print(data['On Hand'].mean())
#4 Filter to A
InventoryA = data[data['Supplier'] =='A']
print(InventoryA) 

#5 add on hand ratio
InventoryA['OnHandRatio'] = InventoryA['On Hand'] / InventoryA['Annual Demand']
print(InventoryA)

#6 Set up avg cost per sku
InventoryA = InventoryA[['Item SKU','Cost per Unit ($)']]
InventoryA = InventoryA.groupby('Item SKU',as_index=False,group_keys=True)['Cost per Unit ($)'].mean()
print(InventoryA)

File: Refresherquizmod1.R
#cars=as_tibble(mtcars)
#str(cars)
diamonddata = diamonds
diamonddata
#1
nrow(diamonddata)
#2
ncol(diamonddata)
#3
ggplot(diamonddata, aes(x=carat, y=price)) + geom_point()
#4
#ggplot(diamonddata, aes(x=carat, y=price,color=cut)) + geom_point()
#5
ggplot(diamonddata, aes(x=carat, y=price)) + geom_point()+ facet_grid(color ~ .)
ggplot(diamonddata, aes(x=carat, y=price)) + geom_point()+ facet_grid(cut ~ .)
#6
Inventory = read_csv("/Users/gregwolcott/Documents/UNCW/BAN 502 - Predictive Analytics/Module 1/InventoryData.csv")
str(Inventory)
#7
mean(Inventory[["On Hand"]])
#8
InventoryA = data_frame(filter(Inventory, Supplier=='A'))
nrow(InventoryA)
#9
InventoryA = mutate(InventoryA, OnHandRatio = `On Hand` / `Annual Demand`)
str(InventoryA)
#10
Newdata <- InventoryA[,colnames(InventoryA)[c(1,4)]]
Newdata

Newdata %>%
  group_by(`Item SKU`) %>%
  summarise(mean_cost = mean(`Cost per Unit ($)`))


